src/app.controller.ts

```diff
@@ -15,16 +15,12 @@ export class AppController {
   getBoardStateFromFile(
     @Query('x') x?: number,
     @Query('y') y?: number,
-  ): { gameState: GameState } {
+  ): { gameState: GameState; winner: StoneColor } {
```

Comment:
why don't you add this attribute inside the GameState interface? imo it's part of the state of the game to know whether the game is won or not

Reply:
Ok good idea! I just choose the faster solution to implement. Because updating the game state interface will probably have an impact on the cli program too, so more stuff to work on. But let's try it !

Reply:
It's done(tell me if it's ok for you), and i also made some refactoring to use this new prop when possible.

Reply:
that's great, well done! 🙂